orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -475,1103 +131,118 @@ Feature: User can specify that a value is equalTo a required value
-    Then the following data should be generated:
-      | foo     |
-      | null    |
-      | <value> |
-    Examples:
-      | value                    |
-      | "a"                      |
-      | 2018-01-01T00:00:00.000Z |
-
-  Scenario: 'EqualTo' run against a contradicting 'lessThan' should only generate null
-    Given there is a field foo
-    And foo is equal to 3
-    And foo is less than 3
-    Then the following data should be generated:
-      | foo  |
-      | null |
-
-  Scenario: 'EqualTo' run against a contradicting not 'lessThan' should only generate null
-    Given there is a field foo
-    And foo is equal to 1
-    And foo is anything but less than 2
-    Then the following data should be generated:
-      | foo  |
-      | null |
-
-### lessThanOrEqualTo ###
-
-  Scenario: 'EqualTo' run against a non contradicting 'lessThanOrEqualTo' is successful
-    Given there is a field foo
-    And foo is equal to 1
-    And foo is less than or equal to 2
-    Then the following data should be generated:
-      | foo  |
-      | null |
-      | 1    |
-
-  Scenario: 'EqualTo' run against a non contradicting not 'lessThanOrEqualTo' is successful
-    Given there is a field foo
-    And foo is equal to 2
-    And foo is anything but less than or equal to 1
-    Then the following data should be generated:
-      | foo  |
-      | null |
-      | 2    |
-
-  Scenario: Not 'equalTo' run against a non contradicting 'lessThanOrEqualTo' is successful
-    Given there is a field foo
-    And foo is anything but equal to 3
-    And foo is less than or equal to 2
-    And foo is in set:
-      | -1       |
-      | 0        |
-      | 2        |
-      | 2.000001 |
-      | 3        |
-    Then the following data should be generated:
-      | foo  |
-      | null |
-      | -1   |
-      | 0    |
-      | 2    |
-
-  Scenario Outline: 'EqualTo' a non-numeric value with 'lessThanOrEqualTo' should be successful
-    Given there is a field foo
-    And foo is equal to <value>
-    And foo is less than or equal to 1
-    Then the following data should be generated:
-      | foo     |
-      | null    |
-      | <value> |
-    Examples:
-      | value                    |
-      | "a"                      |
-      | 2018-01-01T00:00:00.000Z |
-
-  Scenario: 'EqualTo' run against a contradicting 'lessThanOrEqualTo' should only generate null
-    Given there is a field foo
-    And foo is equal to 3
-    And foo is less than or equal to 2
-    Then the following data should be generated:
-      | foo  |
-      | null |
-
-  Scenario: 'EqualTo' run against a contradicting not 'lessThanOrEqualTo' should only generate null
+  Scenario: 'EqualTo' run against a contradicting 'greaterThan' should generate no data
     Given there is a field foo
     And foo is equal to 1
-    And foo is anything but less than or equal to 1
-    Then the following data should be generated:
-      | foo  |
-      | null |
+    And foo is greater than 2
 
 ### granularTo ###
```

Comment:
If we're checking numeric granularity we might as well also have tests for datetime granularity